src/schemas/user.schema.js

```diff
@@ -2,15 +2,57 @@ var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
 var userSchema = new Schema({
-  name: {
+  Id: {
     type: String,
     required: true
   },
+  Fname: {
+    type: String,
+    required: true
+  },
+  Mname: {
+    type: String,
+    required: false
+  },
+  Lname: {
+    type: String,
+    required: true
+  },
+  Bday: {
+    type: String,
+    required: true
+  },
+  Gender: {
+    type: String,
```

Comment:
@melvinpurio  you can use `enum` here.
enum is a list of values that are allowed for specific property.
for example `['male', 'female']`

so this will be ` {type: String, required: true, enum: ['male', 'female']}`

Reply:
Ok copy